src/Nexway/SetupManager/Util/Processor/Action/Upsell/Create.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Nexway\SetupManager\Util\Processor\Action\Upsell;
+
+use Nexway\SetupManager\Util\Processor\AbstractAction;
+/**
+ * @category    Nexway
+ * @package     Nexway_SetupManager
+ * @author      Marcin Tasak <mtasak@nexway.com>
+ * @copyright   Copyright (c) 2015, Nexway
+ */
+class Create extends AbstractAction
+{
+    /**
+     * @return bool
+     */
+    protected function _create()
+    {
+        $this->getParameters()->setModel('nexway_upsell/upsell');
+
+        $data = $this->getParameters()->getData(self::F_DATA);
+
+        $data['upsell_products'] = $data['source_product_id'] . '::' . $data['target_product_id'];
+
+
+        /** @var Nexway_Upsell_Model_Upsell $upsellModel */
+        $upsellModel    = \Mage::getModel('nexway_upsell/upsell');
+
+        /** @var Nexway_Upsell_Model_Options $optionsModel */
+        $optionsModel   = \Mage::getModel('nexway_upsell/options');
+
+        $data = $optionsModel->parseData($data);
+        $upsellModel->setData($data);
+        $upsellModel->save();
+
+        $upsellId = $upsellModel->getId();
+        $upsellModel->setUpsellId($upsellId);
```

Comment:
I don't know why, but upsell model requires upsell_is field for validation. I simplified logic from controller, but i hadn't time to refactor whole validation and save process.